### PR TITLE
Flake8 fixes

### PR DESCRIPTION
**Pull request by tobinjt**
### Motivation for changes:

Fix PEP8 warnings emitted by flake8.

### Detailed changes:

- Lots of small style fixes.

### Addressed issues:

- Fixes # .

### Config usage if relevant (new plugin or updated schema):
```
paste_config_here
```
### Log and/or tests output (preferably both):
```
=========================================================== 1 failed, 1046 passed, 26 skipped, 4 xfailed, 5 pytest-warnings in 418.71 seconds ============================================================
```

Test failure is in test_filesystem.py and was failing before these commits.

### CODE REVIEW

**Comment by liiight** [comment]
can you resync this please?

**Comment by tobinjt** [comment]
Resynced, thanks.

File: flexget/plugins/metainfo/imdb_lookup.py [hunk 1]
```diff
@@ -403,7 +404,8 @@ def lookup(self, entry, search_allowed=True, session=None):
                 log.exception(e)
             raise plugin.PluginError('Invalid parameter: %s' % entry['imdb_url'], log)
 
-        for att in ['title', 'score', 'votes', 'year', 'genres', 'languages', 'actors', 'directors', 'writers', 'mpaa_rating']:
+        for att in ['title', 'score', 'votes', 'year', 'genres', 'languages', 'actors', 'directors', 'writers',
+                    'mpaa_rating']:
             log.trace('movie.%s: %s' % (att, getattr(movie, att)))
```
Comment (cvium) [inline flexget/plugins/metainfo/imdb_lookup.py]: Why is stuff like this not fixed? It should be changed to `log.trace('movie.%s: %s', att, getattr(movie, att))`

Comment (liiight) [inline flexget/plugins/metainfo/imdb_lookup.py]: maybe this is outside the scope of this pr? i'm fine with merging this as is

Comment (tobinjt) [inline flexget/plugins/metainfo/imdb_lookup.py]: I'm only trying to fix warnings from flake8, other fixes are out of scope for this set of changes.

Is there a list of best practices or alternatively old practices that should be changed?  That would be useful for contributors.

Thanks,

Comment (liiight) [inline flexget/plugins/metainfo/imdb_lookup.py]: @cvium meant that logger should interpolate strings on its own, instead of passing them as an interpolated string.
